src/job_consumer/utils_azure_queue.py

```diff
@@ -9,25 +9,4 @@
 
 
 def get_queue_client():
-    if settings.AZURITE_QUEUE_CONNECTION_STRING:
-        queue_service_client = QueueServiceClient.from_connection_string(
-            settings.AZURITE_QUEUE_CONNECTION_STRING
-        )
-
-        # create queue for the first time
-        try:
-            queue_service_client.create_queue(settings.JOB_QUEUE_NAME)
-        except ResourceExistsError:
-            pass
-        
-        queue_client = queue_service_client.get_queue_client(settings.JOB_QUEUE_NAME)
-
-    else:
-        credentials = WorkloadIdentityCredential()
-        queue_client = QueueClient(
-            credential=credentials,
-            account_url=settings.QUEUE_ACCOUNT_URL,
-            queue_name=settings.JOB_QUEUE_NAME,
-        )
-
-    return queue_client
\ No newline at end of file
+    return settings.QUEUE_CLIENT
```

Comment:
Ik vraag me af of de functie `get_queue_client()` nog nodig is. Wordt deze functie op meerdere plaatsen gebruikt? En kunnen we daar dan direct `settings.QUEUE_CLIENT` aanroepen?